ppr-ui/src/composables/useInputRules.ts

```diff
@@ -6,6 +6,13 @@ export const useInputRules = () => {
     ]
   }
 
+  // check if entire value has empty spaces
+  const isEmpty = (): Array<Function> => {
+    return [
+      v => !/^\s+$/g.test(v) || 'Invalid spaces'
+    ]
+  }
+
```

Comment:
I believe Tracey posted she preferred to not have validations? I could be wrong but that was last i heard.

Reply:
Yea, I saw it after and wanted to rework it today.

Reply:
Removed validation for empty spaces.